core/frontend/src/App.vue

```diff
@@ -269,6 +279,29 @@ export default Vue.extend({
     changeBackendStatus(backend_offline: boolean): void {
       this.backend_offline = backend_offline
     },
+    convertGitDescribeToUrl(git_describe: string): string {
+      const user = 'bluerobotics'
+      const repository = 'companion-docker'
+      const project_url = `https://github.com/${user}/${repository}`
+
+      // Local development version, pointing to root page
+      if (git_describe.endsWith('-dirty')) {
+        return project_url
+      }
+
+      // Show tag release page
+      if (git_describe.startsWith('tags')) {
+        const match = /tags\/(?<tag>|.*)-\d-.*/gm.exec(git_describe)
+        if (match && match.groups?.tag) {
+          const { tag } = match.groups
+          return `${project_url}/releases/tag/${tag}`
+        }
+      }
+
+      // Show git source files page for commit
+      const hash = git_describe.slice(git_describe.length - 7)
+      return `${project_url}/tree/${hash}`
+    },
```

Comment:
The code is good. I just few it should be on some other file, not App.vue, so it don't get bloated with functional logic.

Reply:
Done.